### PR TITLE
Update CODEOWNERS > narrow down scopes

### PR DESCRIPTION
**Pull request by patricebender**
With this change, I want to achieve that people feel more obligated to review a particular change.
This can be achieved by restricting the code ownership to specific areas. E.g. @vobu @gregorwolf @sebastianesch might be interested in `postgres` while not being interested in changes in e.g. `cqn4sql`.

Our product owners @stewsk @johannes-vogel and @danjoa are still owning everything and hence are able to approve changes in all areas.

If you have suggestions how to improve my initiative, please let me know.

---

@David-Kunz  @sjvans

would you like to be a code owner in a particular area?

### CODE REVIEW

**Comment by BobdenOs** [comment]
Probably would be good to have code owners for the `.md` files (e.g. [CONTRIBUTING.md](https://github.com/cap-js/cds-dbs/blob/main/CONTRIBUTING.md)).

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,25 @@
...
+### Product owner own everything
+* @danjoa @johannes-vogel @stewsk
+
+db-service/lib/cqn4sql.js @patricebender
+db-service/test/cqn4sql/ @patricebender
+
+db-service/lib/infer/**.js @patricebender
+db-service/test/cds-infer/ @patricebender
+
+db-service/lib/InsertResults.js @BobdenOs
```
Comment (patricebender) [inline .github/CODEOWNERS:10]:
```suggestion
db-service/lib/InsertResults.js @BobdenOs
db-service/lib/converters.d.ts @BobdenOs
```

**Review by gregorwolf — APPROVED** [review]
LGTM

**Comment by sjvans** [comment]
> would you like to be a code owner in a particular area?

i'm fine for now. if absolutely necessary, i can add myself again. ;)

**Pull request merged**